File: app/src/main/java/com/example/requestapp/ui/fragment/availableTasksFragment/AvailableTasksFragmentContract.java
package com.example.requestapp.ui.fragment.availableTasksFragment;

import com.example.requestapp.model.Task;

import java.util.List;

public interface AvailableTasksFragmentContract {

    interface Presenter {
    }

    interface View {

        void setListMedium(List< Task > task);

        void setListHigh(List< Task > task);

        void setListLow(List< Task > task);

    }
}

File: app/src/main/java/com/example/requestapp/ui/fragment/availableTasksFragment/AvailableTasksFragmentContract.kt
package com.example.requestapp.ui.fragment.availableTasksFragment

import com.example.requestapp.model.Task

interface AvailableTasksFragmentContract {

    interface Presenter

    interface View {
        fun setListMedium(task: List<Task?>?)
        fun setListHigh(task: List<Task?>?)
        fun setListLow(task: List<Task?>?)
    }
}